bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts

```diff
@@ -456,7 +473,15 @@ export class KtbSequenceViewComponent implements OnDestroy {
           this.selectSequence({ sequence: sequence });
           this.loadTraces(sequence, undefined, sequenceExecution.scope.stage);
         } else {
-          this.dataService.loadUntilRoot(sequenceExecution.scope.project, sequenceExecution.scope.keptnContext);
+          this.dataService
+            .loadUntilRoot(sequenceExecution.scope.project, sequenceExecution.scope.keptnContext)
+            .subscribe((sequences) => {
+              const newSequence = sequences.find((seq) => seq.shkeptncontext === sequenceExecution.scope.keptnContext);
+              if (newSequence) {
+                this.selectSequence({ sequence: newSequence });
+                this.loadTraces(newSequence, undefined, sequenceExecution.scope.stage);
+              }
+            });
```

Comment:
There should be a way to get rid of the double subscribe? By chaining with RXJS operators...